### DISCUSSION TITLE
Can't delegate or send

### DISCUSSION THREAD

**Discussion by julienwagener** [ping-pub/explorer#469] [Q&A]
Hello,
I connected my wallet through Ledger.
Then deposited money.
But then it seems I cant delegate and/or send?

I have the following issue:
![Screenshot 2023-07-04 at 14 45 55](https://github.com/ping-pub/explorer/assets/39762276/6acabf4f-33f0-4d46-b1eb-8e71712f1825)

Can someone please help?
Thanks

**Comment by liangping** [comment]
You got a wrong key.

**Reply by julienwagener** [reply]
Hello,
Thanks for taking the time to come back to me.
I agree that the message suggest I have a wrong public key. But unfortunately I am using the secret app on ledger and I am connected my wallet though ledgerUSB, therefore I cannot see/change/edit the key as they are inside my ledger.
Any other idea?
Thanks
Julien
